app/views/student_requests/index.html.haml

```diff
@@ -3,6 +3,7 @@
 %table#movies
   %thead
     %tr
+      %th Request ID
```

Comment:
don't show request id to student. Its internal to force request. It shouldn't be exposed outside and also there is no need.